src/rHelpPanel.ts

```diff
@@ -1,5 +1,10 @@
+/* eslint-disable @typescript-eslint/restrict-template-expressions */
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/* eslint-disable @typescript-eslint/no-unsafe-return */
 
-import * as vscode from 'vscode';
+import { commands, window, QuickPickItem, Uri, Webview, WebviewPanel, WebviewOptions, WebviewPanelOnDidChangeViewStateEvent, ViewColumn } from 'vscode';
```

Comment:
Is there a performance benefit from importing each item explicitly or is this just a stylistic choice?

Reply:
It is not alerted by ESLint, but it is useful to refactoring based on the vscode API.
In this case, we can confirm  `rHelpPanel.ts` manage the Webview API related function.
If someone implements a related function in another file, we should combine that in this file or implement super-class